plugins/extractors/kafka/kafka.go

```diff
@@ -65,8 +89,20 @@ func (e *Extractor) Init(ctx context.Context, config plugins.Config) (err error)
 		return err
 	}
 
+	// create default dialer
+	dialer := kafka.DefaultDialer
```

Comment:
nit: avoid modifying the global instance.

```suggestion
	dialer := &kafka.Dialer{
		Timeout:   10 * time.Second,
		DualStack: true,
	}
```